TAF/testScenarios/functionalTest/V2-API/core-metadata/provisionwatcher/GET-Positive.robot

```diff
@@ -0,0 +1,113 @@
+*** Settings ***
+Resource     TAF/testCaseModules/keywords/common/commonKeywords.robot
+Suite Setup  Run Keywords  Setup Suite
+...                        AND  Run Keyword if  $SECURITY_SERVICE_NEEDED == 'true'  Get Token
+Suite Teardown  Run Keyword if  $SECURITY_SERVICE_NEEDED == 'true'  Remove Token
+Force Tags
+
+*** Variables ***
+${SUITE}          Core Metadata Provision Watcher GET Positive Test Cases
+${LOG_FILE_PATH}  ${WORK_DIR}/TAF/testArtifacts/logs/core-metadata-provisionwatcher-get-positive.log
+${api_version}    v2
+
+*** Test Cases ***
+# /provisionwatcher/all
+ProWatcherGET001 - Query all provision watcher
+    Given Create Multiple Provision Watchers
+    When Query All Provision Watchers
+    Then Should Return Status Code "200"
+    And Should Have Content-Type "application/json"
+    And Validate Response Schema
```

Comment:
Instead of `Validate Response Schema`, check the number of provision watchers.

Reply:
Added.